raiden/transfer/token_network.py

```diff
@@ -327,14 +319,6 @@ def state_transition(
         iteration = handle_channel_close(
             token_network_state, state_change, block_number, block_hash
         )
-    elif type(state_change) == ActionChannelSetFee:
```

Comment:
Why is this removed, we remove the Channel set fee action? maybe i miss understood the point of the change

Reply:
yes, I mean, it never existed